libraries/adaptive-expressions-ie11/package.json

```diff
@@ -31,17 +31,15 @@
     "@babel/preset-typescript": "^7.6.0",
     "@babel/runtime": "^7.6.3",
     "babel-loader": "^8.0.6",
-    "copyfiles": "2.4.0",
+    "clean-webpack-plugin": "^3.0.0",
     "terser-webpack-plugin": "^4.2.3",
     "ts-loader": "^7.0.5",
     "typescript": "3.5.3",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.12"
   },
   "scripts": {
-    "build:webpack": "webpack",
-    "build:copyfiles": "copyfiles -u 3 node_modules/adaptive-expressions/lib/**/*.d.ts dist",
```

Comment:
Note: the generated declaration file is:

```typescript
export * from 'adaptive-expressions';
```

Which the Typescript compiler will properly resolve. No need to copy over all the generated declaration files.